lib_ux_sync/src/ux_sync.c

```diff
@@ -1,10 +1,24 @@
 #ifdef HAVE_NBGL
 
 #include "ux_sync.h"
+#include "os_helpers.h"
 
 static ux_sync_ret_t g_ret;
 static bool          g_ended;
 
+void generic_content_action_callback(int token, uint8_t index, int page)
+{
+    UNUSED(index);
+    UNUSED(page);
+    if (token == FIRST_USER_TOKEN) {
+        g_ret = UX_SYNC_RET_APPROVED;
+    }
+    else if (token == FIRST_USER_TOKEN + 1) {
+        g_ret = UX_SYNC_RET_REJECTED;
+    }
```

Comment:
g_ret might end up unmodified, is this expected ?
If so write a comment to explain the case where it happens

Reply:
Not sure we want this to happen... I might end up just using an "else" :thinking:

Reply:
@fbeutin-ledger new PR on master with changes according to your request : https://github.com/LedgerHQ/ledger-secure-sdk/pull/705